package.json

```diff
@@ -2,19 +2,20 @@
   "name": "aws-polly-voices",
   "version": "0.0.0-development",
   "description": "Helper module to get AWS polly voices in an idiomatic way",
-  "main": "dist/index.js",
+  "main": "dist/index.ts",
```

Comment:
I don't think we should serve the **ts** file because a simple nodejs app doesn't support it out of the box. Instead, we should add the `types` property.

Reply:
Thanks for pointing this out! This change was unintended, I reverted this. The build process has of course not been touched.

I've also added a `types` property pointing to a new custom `index.d.ts` type declaration file. However I have to point a few things out here:
* this is my first attempt wrting a custom file declaration file for a module; I had to do some research first to get it right - so please feel free to double check
* babel's build process won't do any type checking; I've added a new `check-types` command to use the Typescript compiler manually without emmiting output
* I suggest to create a new issue to automatically apply type-checking to a commit-hook

Other than that: Sorry for the long delay 🙏